server/integration/feature-pack/src/main/resources/content/bin/agent.conf

```diff
@@ -0,0 +1 @@
+jolokia=port=8778
```

Comment:
With this conf, jolokia will bind only to ```127.0.0.1``` which is a good default security-wise. Is this enough for Openshift to operate? 

Also, Wildfly has cmd line flag ```-b``` flag to control the binding of application level services, and ```-bmanagement``` to control binding of the admin console. Wondering if it's worth it to provide a switch to control the jolokia binding or piggyback in the ```-bmanagement``` one, as an easier way of getting it running quickly without providing a custom config via --jmx.

Reply:
This file needs to be overridden for OpenShift :)

OpenShift needs different settings and certificates. Here's an example: https://github.com/infinispan/infinispan-openshift-templates/pull/6/files#diff-edf8fc0cedef8b07d126d61e0799548aR74